packages/react-widgets/src/widgets/TimeSeriesWidget.js

```diff
@@ -18,6 +18,10 @@ import { columnAggregationOn } from './utils/propTypesFns';
 import useWidgetFetch from '../hooks/useWidgetFetch';
 import WidgetWithAlert from './utils/WidgetWithAlert';
 
+function getUTCTime (date) { 
```

Comment:
I'm a bit worried about this fix.
I think at least the name of this function is misleading, as it's not returning UTC time. 
`Date.getTime` already returns a UTC ms timestamp, if we apply an offset to it we convert it to a "local timestamp".

If I understand correctly, the problem is that filter values from the widget are UTC, whereas the feature values from the source are manipulated as local dates (is it always the case?), so the filter fails.
Maybe a better place to apply a fix is at the filter level itself "https://github.com/CartoDB/carto-react/blob/57821621b1dc179327a42f0054d930ecf04b1173/packages/react-core/src/filters/FilterTypes.js#L16-L23" or in how `timeWindow` values are created?

Sorry @moicalcob, just trying to understand in more depth. TZ stuff is notoriously confusing so we need to take extra care that it will work everywhere in the world/anytime/with every dataset.
Maybe @padawannn you have some insight here too?